$OpenBSD: patch-configure,v 1.2 2004/04/23 02:17:50 dons Exp $
Let alex know about the new archs Haskell runs on.

--- configure.orig	Wed Aug 13 23:36:18 2003
+++ configure	Thu Apr 22 14:05:01 2004
@@ -1506,6 +1506,15 @@
         HostVendor_CPP='unknown'
         HostOS_CPP='freebsd'
         ;;
+amd64*-unknown-openbsd*)
+	HostPlatform=x86_64-unknown-openbsd
+	TargetPlatform=x86_64-unknown-openbsd
+	BuildPlatform=x86_64-unknown-openbsd
+        HostPlatform_CPP='x86_64_unknown_openbsd'
+        HostArch_CPP='x86_64'
+        HostVendor_CPP='unknown'
+        HostOS_CPP='openbsd'
+        ;;
 hppa1.1-hp-hpux*)
 	HostPlatform=hppa1.1-hp-hpux  # canonicalise for our purposes (hack)
 	TargetPlatform=hppa1.1-hp-hpux
@@ -1714,6 +1723,15 @@
         HostArch_CPP='sparc'
         HostVendor_CPP='sun'
         HostOS_CPP='solaris2'
+        ;;
+sparc-unknown-openbsd*)
+      HostPlatform=sparc-unknown-openbsd
+      TargetPlatform=sparc-unknown-openbsd
+      BuildPlatform=sparc-unknown-openbsd
+        HostPlatform_CPP='sparc_unknown_openbsd'
+        HostArch_CPP='sparc'
+        HostVendor_CPP='unknown'
+        HostOS_CPP='openbsd'
         ;;
 *)
         echo "Unrecognised platform: $HostPlatform"
